tests/redirects/map_globalconf.py

```diff
@@ -985,6 +985,7 @@
              re.compile(r'/en-US/firefox/[\d\.]+/releasenotes/'),
              status_code=requests.codes.found),
     url_test('/firefox/brand/', '/styleguide/'),
+    url_test('/styleguide/identity/mozilla/logo-prototype/', '/styleguide/'),
```

Comment:
Nit: instead of bundling this up in redirects that we're ported over from mcom-tests, it would be nice to keep this separate along with a reference to the bug number?